wpilib/src/lib.rs

```diff
@@ -4,11 +4,24 @@
 // <https://www.gnu.org/licenses/> for a copy.
 
 #![feature(concat_idents)]
```

Comment:
Hmm. Is this feature still used in this crate directly? I think this only applies to `wpilib-sys`, unless there's a macro invocation somewhere I haven't noticed.

Reply:
The use of resource_type!() from wpilib-sys in wpilib means it also requires the feature.